seccomp.go

```diff
@@ -0,0 +1,17 @@
+package seccomp
```

Comment:
Shouldn't this be conditional compiled with the seccomp tag? As written, a system built without Seccomp can still report Seccomp enabled.

Reply:
Yeah I agree, fixed it with the next commit.